datumaro/plugins/data_formats/datumaro/base.py

```diff
@@ -317,6 +322,14 @@ def _load_annotations(item):
                     )
                 )
 
+            elif ann_type == AnnotationType.hash_key:
+                hash_key = np.asarray(hash_key, dtype=np.uint8)
+                loaded.append(
+                    HashKey(
+                        hash_key=np.array(hash_key),
+                    )
+                )
+
```

Comment:
You don't need this because `parse_hashkey_file()` will load hash_key from the separate file.